pkg/kubectl/cmd/util/shortcut_restmapper.go

```diff
@@ -124,17 +124,57 @@ func (e ShortcutExpander) AliasesForResource(resource string) ([]string, bool) {
 		}
 		return aliases, true
 	}
-	expanded := expandResourceShortcut(schema.GroupVersionResource{Resource: resource}).Resource
+	expanded := e.expandResourceShortcut(schema.GroupVersionResource{Resource: resource}).Resource
 	return []string{expanded}, (expanded != resource)
 }
 
+// resourceShortcuts represents a structure that holds the information how to
+// transition from resource's shortcut to its full name.
+type resourceShortcuts struct {
+	shortForm schema.GroupResource
+	longForm  schema.GroupResource
+}
+
+// getShortcutMappings returns a hardcoded set of tuples.
+// First the list of potential resources will be taken from the instance variable
+// which holds the anticipated result of the discovery API.
+// Next we will fall back to the hardcoded list of resources.
+// Note that the list is ordered by group priority.
+// TODO: Wire this to discovery API.
+func (e ShortcutExpander) getShortcutMappings() ([]resourceShortcuts, error) {
+	res := []resourceShortcuts{
+		{
+			shortForm: schema.GroupResource{Group: "storage.k8s.io", Resource: "sc"},
+			longForm:  schema.GroupResource{Group: "storage.k8s.io", Resource: "storageclasses"},
+		},
+	}
+
+	// append hardcoded short forms at the end of the list
+	for short, long := range kubectl.ShortForms {
```

Comment:
alright I will change the type of `kubectl.ShortForms`, adjust the existing code and then it would be nice if you could fill the groups info